packages/iTwinUI-react/src/core/Slider/Track.tsx

```diff
@@ -80,7 +81,17 @@ export const Track = (props: TrackProps) => {
               {shouldDisplaySegment(index, trackDisplayMode) ? (
                 <div
                   className='iui-slider-track'
-                  style={{ left: `${leftPercent}%`, right: `${rightPercent}%` }}
+                  style={{
+                    ...(orientation === 'horizontal'
+                      ? {
+                          left: `${leftPercent}%`,
+                          right: `${rightPercent}%`,
+                        }
+                      : {
+                          top: `${rightPercent}%`,
```

Comment:
Same as in Thumb, would be nice to rename to more generic name.

Reply:
Done in [https://github.com/iTwin/iTwinUI-react/pull/747#discussion_r931528071](here)